content/en/developers/ide_plugins/vscode/_index.md

```diff
@@ -91,44 +101,105 @@ The **View in VS Code** feature provides a link from Datadog directly to your so
 
-{{< img src="/developers/ide_plugins/vscode/static-analysis-issue.png" alt="A static analysis rule violation and recommended fix." style="width:100%;" >}}
+[Static Analysis][14] supports scanning for many programming languages. For a complete list, see [Static Analysis Rules][15]. For file types belonging to supported languages, issues are shown in the source code editor with the VS Code inspection system, and suggested fixes can be applied directly.
 
-Additionally, all issues detected by this feature are listed in the standard **Problems** view.
+{{< img src="/developers/ide_plugins/vscode/static_analysis.mp4" alt="Preview of Static Analysis" style="width:100%" video=true >}}
 
 ### Getting started
-When you start editing a source file supported by Static Analysis, the extension checks for `static-analysis.datadog.yml` at your source repository's root. It prompts you to create the file if necessary:
+
+When you start editing a source file, the extension checks for [static-analysis.datadog.yml][22] at your source repository's root. It prompts you to create it if necessary.
 
 {{< img src="/developers/ide_plugins/vscode/static-analysis-onboard.png" alt="A banner for onboarding." style="width:100%;" >}}
 
 Once the configuration file is created, the static analyzer runs automatically in the background.
 
 <div class="alert alert-info">The Static Analysis feature does not require a Datadog account, as source files are analyzed locally.</div>
 
-## Feedback
+## License
+
+Please read this [End-User License Agreement][eula] carefully before downloading or using the Datadog Visual Studio Code Extension.
+
+## Data and telemetry
+
+Datadog anonymously collects information about your usage of this IDE, including how you interact with it, whether errors occurred while using it, and what caused those errors, in accordance with the [Datadog Privacy Policy][datadog_privacy_policy] and Datadog's [VS Code extension EULA][eula].
+
+If you don't wish to send this data to [Datadog][datadog], you can opt out at any time in the extension settings: `Datadog > Telemetry > Setup > Enable Telemetry`.
+
+<div class="alert alert-info">The Datadog extension also honors the [VS Code telemetry][vs_code_telemetry] telemetry setting.</a>
+
+## Help and feedback
+
+To share your feedback, email [team-ide-integration@datadoghq.com][feedback_email] or create an issue in the extension's [public repository][public_repo].
 
-To share your feedback, email [team-ide-integration@datadoghq.com][9].
+Check out the [issues][known_issues] section to discover known issues.
 
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /continuous_testing/
-[2]: https://www.datadoghq.com/
+[datadog]: https://www.datadoghq.com/
```

Comment:
```suggestion
[datadog]: https://www.datadoghq.com/
```

Hi @robertohuertasm, can you please update these relative link references to the correct consecutive numbers to be consistent with the docs style guide? Thank you!